src/misc/string_utils.cpp

```diff
@@ -263,12 +263,14 @@ std::string strip_word(std::string& line)
 	return word;
 }
 
-void strip_punctuation(std::string &str)
+void strip_punctuation(std::string& str)
 {
-	str.erase(std::remove_if(str.begin(),
-	                         str.end(),
-	                         [](unsigned char c) { return std::ispunct(c); }),
-	          str.end());
+	auto punct_chars = {
+	        '\t', '?', '/', '\\', '|', '.', ',', ';', ':', '+', '=', '[', ']', '<', '>', '"', '\''};
+	auto is_punct = [punct_chars](unsigned char c) {
+		return std::count(punct_chars.begin(), punct_chars.end(), c) > 0;
```

Comment:
`std::count` is not the best fit here; although it works, it muddies the intent a bit.

[std::find](https://en.cppreference.com/w/cpp/algorithm/find.html) is the perfect solution, can you please use that?

Reply:
> Or `contains()` in `support.h`

Forgot about that, that's the winner, yeah 😄